tsconfig.json

```diff
@@ -12,7 +12,8 @@
         "sourceMap": true,
         "rootDirs": [
             "src",
-            "third-party"
+            "third-party",
+            "unit-test"
```

Comment:
I don't see this used anywhere here. Revert the file changes

Reply:
Right, if it's not used I should remove it. I understand that Matt had tried to do some work so our test are less brittle and tightly coupled with the VS Code environment. But there is concern about changing too much at once. Here's a potential approach to address this in baby steps. 

1. Have two test folders, the original one `src/test` (integ tests) and `unit-test` 
2. Change `package.json` to add `unit-test` and have the `test`  target call the unit tests then the integ tests.
3. Each time we address an issue, we have the opportunity to move the non-tightly coupled tests into `unit-test` and perhaps perhaps convert some of the code into pure functions with associated unit tests.

The result is that we will call the faster `npm run unit-test` more frequently than `npm run test` as we are coding. This will also promote less brittle/more re-usable coding practices. The code tested by unit tests could likely be shared with the Cloud9 team or others using node that are trying to do similar things.

Reply:
I like the spirit of this idea, and we can work towards that. It is worthwhile seeding a change like that in a dedicated PR with an initial candidate test that ensures the configurations/pipelines/local development are working end-to-end.